lib/tizen_plugins.dart

```diff
@@ -66,6 +66,16 @@ class TizenPlugin extends PluginPlatform implements NativeOrDartPlugin {
   final String dartPluginClass;
   final String fileName;
 
+  TizenPlugin copyWith({Directory directory}) {
```

Comment:
How about changing to `copyTo` and including the directory copying. 
```dart
TizenPlugin copyTo(Directory destination) {
  if (destination == null) {
    throw Exception('destination is null');
  }
  copyDirectory(directory, destination);
  return TizenPlugin(
    ....
}
```

Reply:
Actually it was an adaptation of the "copyWith" pattern widely used when developing Flutter apps.